# Mecánica de fluidos - Práctico 1

## Problema 1
Con referencia  a la figura y despreciando el rozamiento entre el pistón A y el cilindro que contiene el gas, determinar la presión manométrica en B en metros de columna de agua. Suponer el peso específico del gas 0,56 kg/m³.

![tp1-01.png](tp1-01.png)

### Solución
El área de $A = A_a = \pi r^2 = \pi \, (30 \, m)^2 $

In [3]:
import math
r = 60 / 2 # m
A_a = math.pi * r**2 # m^2
print('A_a =',round(A_a,2),'m^2')

A_a = 2827.43 m^2


La presión relativa bajo A es
$$ p_a = \frac{P_A}{A_a} $$

In [5]:
P_A = 1600 * 1000 # kgf peso del pistón
p_a = P_A / A_a # presión relativa bajo A
print('p_a =',round(p_a,2),'kgf/m^2')

p_a = 565.88 kgf/m^2


Para un gas ideal en condiciones isotérmicas
$$ p v = p_1 v_1 = R \qquad \qquad [1] $$
ahora como
$$ \gamma = g \delta \quad \Rightarrow \frac{1}{\delta} = \frac{g}{\delta} $$ y como 
$$ v = \frac{1}{\delta} \quad \Rightarrow \quad v = \frac{q}{\gamma} \qquad [2] $$
reemplazando [2] en [1] resulta:
$$ p \frac{g}{\gamma} = p_1 \frac{g_1}{\gamma_1} = R \qquad [3] $$
Suponiendo que el intervalo de elevación no es lo suficientemente grande como para que introduzca una variación significativa en $g$, de manera que $g$ puede considerarse como una constante. Luego, dividiendo por $g$ en [3]
$$ \frac{g}{\gamma} = \frac{p_1}{\gamma_1} = \frac{R}{g} = C \qquad [4] \qquad \therefore \quad \gamma = \frac{P}{C} \qquad [5] $$
Utilizando [5], puede expresarse la ecuación diferencial básica como sigue:
$$ \frac{dP}{dz} = \gamma = \frac{P}{C} $$
Separando variables e integrando entre $p_1$ y $p$ y entre $z_1$ y $z$, se tiene
## $$ \int_{p_1}^P \frac{dP}{P} = \int_{z_1}^z \frac{dz}{C} \qquad \Rightarrow \qquad \left. \ln p \right|_{p_1}^p = \left. \frac{z}{C} \right|_{z_1}^z \qquad \Rightarrow \qquad \ln \frac{p}{p_1} = \frac{1}{C} (z - z_1) $$
despejando $p$ y utilizando la ecuación [4] $p_1/ \gamma_1 = C$
$$ \frac{p}{p_1} = e^{(\gamma_1 / p_1) (z-z_1)} $$
## $$ p = p_1 \, e^{(\gamma_1 / p_1) (z-z_1)} \qquad [7] $$
Como $p_1 = p_a = 565,88 \, kgf/m^2$ y $z_1 = 0$, $z = 70 \, m$ reemplazando en [7] resulta

In [10]:
p_1 = p_a
gamma = 0.56 # kgf/m^3
z = 70
z_1 = 0 # m
p_b = p_1 * math.e**(gamma/p_1*(z-z_1))
print('p_b =',round(p_b,2),'kgf/m^2')

p_b = 606.47 kgf/m^2


### $$ p_{H_2O} = \gamma_{H_2O} \cdot h = 1000 \, kgf/m^3 \cdot 1 \, m = 1000 \, kgf/m^2 = 1 \, mca $$
## $$ p_B = 606,47 \, kgf/m^2 = 0,606 \, mca = 60,64 \, cmca $$

In [ ]:
%reset -f